infrastructure/src/main/resources/templates/errorpages/error.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en"
+      th:replace="~{mopslayout :: html(name='feedback', headcontent=~{:: headcontent}, navigation=~{:: navigation}, bodycontent=~{:: bodycontent})}"
+      xmlns:th="http://www.thymeleaf.org">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Error</title>
+    <th:block th:fragment="headcontent">
+        <style>
+            .container {
+                position: relative;
+                text-align: left;
+                color: white;
+            }
+
+            .top-center {
+                position: absolute;
+                top: 8px;
+                left: 16px;
+                color: black;
+
+            }
+        </style>
```

Comment:
Please DO NOT use custom css to hard code pixel amounts into anything. We want to keep our templates reactive. If you want to do this kind of stuff, you can refer to the Bootstrap documentation and add the proper classes.